apps/web-desmos/src/screens/proposal_details/components/overview/index.tsx

```diff
@@ -26,10 +26,14 @@ const Overview: FC<{ className?: string; overview: OverviewType }> = ({ classNam
   const { classes, cx } = useStyles();
   const { t } = useAppTranslation('proposals');
 
-  const type =
-    R.pathOr('', [0, '@type'], overview.content) === ''
-      ? getProposalType(R.pathOr('', ['@type'], overview.content))
-      : getProposalType(R.pathOr('', [0, '@type'], overview.content));
+  const types: string[] = [];
```

Comment:
Because this is used as a dependency of the `useCallback` hook, I would recommend using `useMemo` here or the callback will be created every time (not only this line but all the types definition)